package.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "cross-polyform",
-  "homepage": "https://PolyPhyHub.github.io/PolyPhy-Website",
```

Comment:
@AsavariA Yes, why are we taking out this description.

Reply:
Because its not the final URL where our site will be deployed, I will update it once we get our domain name and hosting